pkg/aws/ec2_instance.go

```diff
@@ -0,0 +1,85 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/service/ec2"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/Qovery/pleco/pkg/common"
+)
+
+type EC2Instance struct {
+	common.CloudProviderResource
+}
+
+func deleteEC2Instances(ec2Session *ec2.EC2, ec2Instances []EC2Instance) {
+	for _, ec2Instance := range ec2Instances {
+		var instanceIds []*string
+		instanceIds = append(instanceIds, &ec2Instance.Identifier)
```

Comment:
Didn't find a way to inline the creation & instantiation of an array :thinking:

Reply:
oneSlice := []*string{"abc", "def"}

Something like that?

Reply:
Thanks